### ISSUE TITLE
XML structure is probably wrong (and weird at least)

### ISSUE THREAD

**Issue by boekabart** [umpirsky/locale-list#5]
First off: Thank you for this and the country list as well!

I ended up using the json version because the xml of both projects is wrong or at least weird. Take a look:
```xml
<?xml version="1.0" encoding="utf-8"?>
<values>
    <item>
        <id>af</id>
        <![CDATA[Afrikaans]]>
        <value />
    </item>
    <item>
        <id>af_NA</id>
        <![CDATA[Afrikaans (Namibia)]]>
        <value />
    </item>
```
I *presume* that the intent was to have the value inside the `<value>` tag like so:
```xml
<?xml version="1.0" encoding="utf-8"?>
<values>
    <item>
        <id>af</id>
        <value><![CDATA[Afrikaans]]></value>
    </item>
```

The current format is probably technically correct XML, but very hard to parse by most libraries because the `<item>` element mixes both child elements and text; which is unusual.

**Comment by umpirsky** [comment]
Ouch, that does look wrong, thanks!
